bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableOptions.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.dataflow;
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+
+/**
+ * CloudBigtableOptions is an extension of {@link PipelineOptions} the information that {@link
+ * CloudBigtableConfiguration} needs to configure a connection to Cloud Bigtable.
+ */
+public interface CloudBigtableOptions extends PipelineOptions {
+  String getBigtableProject();
```

Comment:
Do these options need validation? Should any of them be marked required?

Can you also add a description of the interface and each of the options:

https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/bdde37492d5f79b3570a543fe6392391023dddec/sdk/src/main/java/com/google/cloud/dataflow/sdk/options/BigQueryOptions.java